examples/ceed/ex1.c

```diff
@@ -53,6 +53,7 @@ CeedScalar TransformMeshCoords(int dim, int mesh_size, CeedVector mesh_coords);
 int main(int argc, const char *argv[]) {
   const char *ceed_spec = "/cpu/self";
   int dim        = 3;           // dimension of the mesh
+  int ncomp      = 3;           // number of field components
```

Comment:
I think we need to be careful here, as 3 is the number of components for the geometry, not for the problem being solved.

Reply:
Right. I thought it could be ambiguous. I have renamed it `ncompx` as it is in the petsc/bps now